datastore/src/main/java/io/spine/server/storage/datastore/DatastoreStorageFactory.java

```diff
@@ -164,22 +161,22 @@ public BoundedContextBuilder configureTenantIndex(BoundedContextBuilder builder)
     createRecordStorage(ContextSpec context, RecordSpec<I, R, ?> spec) {
         checkNotNull(context);
         checkNotNull(spec);
-        StorageConfiguration<I, R> config = configurationWith(context, spec);
-        Optional<CreateStorage<I, R>> custom = customStorages.find(spec);
-        RecordStorage<I, R> result =
+        var config = configurationWith(context, spec);
+        var custom = customStorages.find(spec);
+        var result =
                 custom.map(callback -> callback.apply(config))
```

Comment:
Please join this line to the previous one.

Reply:
Left as-is to preserve `.map` to `.orElse` alignment in IDEA.